client/my-sites/email/controller.js

```diff
@@ -6,6 +6,7 @@ import TitanControlPanelRedirect from 'calypso/my-sites/email/email-management/t
 import TitanManageMailboxes from 'calypso/my-sites/email/email-management/titan-manage-mailboxes';
 import TitanManagementIframe from 'calypso/my-sites/email/email-management/titan-management-iframe';
 import EmailProvidersComparison from 'calypso/my-sites/email/email-providers-comparison';
+import { EmailProvidersInDepthComparison } from 'calypso/my-sites/email/email-providers-in-depth-comparison';
```

Comment:
We should probably `export default` that component so we can import it like the other ones:

```suggestion
import EmailProvidersInDepthComparison from 'calypso/my-sites/email/email-providers-in-depth-comparison';
```